MergerService/Utils/TaskUtils.cs

```diff
@@ -15,161 +14,30 @@ public class TaskUtils : ITaskUtils
         private ILogger _logger;
         private ActivitySource _activitySource;
 
-        public TaskUtils(IConfigurationManager configuration, IHttpRequestUtils httpClient, ILogger<TaskUtils> logger, ActivitySource activitySource)
+        private int _maxAttempts;
+
+        public TaskUtils(int maxAttempts, IConfigurationManager configuration, IHttpRequestUtils httpClient, ILogger<TaskUtils> logger, ActivitySource activitySource)
```

Comment:
why max attempts isnt read from config object?

Reply:
Changed to use configuration.